iModelJsNodeAddon/api_package/ts/src/NativeLibrary.ts

```diff
@@ -1004,6 +1004,10 @@ export declare namespace IModelJsNative {
     public get alias(): string;
     /** The logId. */
     public get logId(): string;
+    /** The time that the write lock expires. Of the form 'YYYY-MM-DDTHH:MM:SS.000Z' in UTC.
+     *  Returns empty string if no write lock expiry time.

```

Comment:
```suggestion
     *  Returns empty string if write lock is not held.

```

Reply:
Technically if the write lock was never released (maybe due to some failed process), the write lock expiry time would not be an empty string but the write lock would also not be held (since expiry time is at some point in the past). Should this be reflected in the comment?

Reply:
The important point for the comment is that the caller must be aware that the string will be empty if the lock is not held. If the string is valid, it's a date string that tells you that the lock was successfully acquired and hasn't yet been released. Whether or not the lock has expired (or was "stolen" by an admin) isn't relevant for this call, I don't think. BTW, even if the lock expires things can still work fine unless/until someone else attempts to acquire the lock, notices it is expired, and  acquires it.